classifiers/nlp/ZeroShotTFClassifier/manifest.yml

```
@@ -0,0 +1,13 @@
+manifest_version: 1
+name: ZeroShotTFClassifier
+kind: classifier
+type: pod
+description: :class:`ZeroShotTFClassifier` wraps tensorflow transformers from huggingface,
+  and performs zero shot learning classification for nlp data.
+author: Jina AI Dev-Team (dev-team@jina.ai)
```

hi, @julianpetrich We'd suggest to put your github account name here. In the future, we plan to provide a GUI for managing all your hub images at one place.

```suggestion
author: julianpetrich
```